standards.iso.org.annotated/iso/19115/resources/transforms/ISO19139/mapping/core.xsl

```diff
@@ -435,7 +435,7 @@
   </xsl:template>
   <xsl:template match="gmi:objectiveOccurance" mode="from19139to19115-3">
     <!-- This element is mis-spelled in the 19115-2 schema -->
-    <xsl:element name="mac:objectiveOccurence">
+    <xsl:element name="objectiveOccurence">
```

Comment:
Should we keep the "mac" prefix?

Reply:
Oh, my mistake, didn't mean to remove the prefix. Fixing now.